README.md

```diff
@@ -6,8 +6,14 @@ Soy el director del departamento de análisis de riesgos de una empresa de banca
 - Se trata de un problema real del que tengo conocimiento, pues es uno de los campos que estoy estudiando en prácticas de empresa.
 - No es necesario ningún servicio ni aparato externo para funcionar, meramente el sistema o servidor donde se albergue la aplicación.
 - La solución requiere una cierta cantidad de lógica de negocio, ya que un mal procesamiento de datos es la base de la que parte este problema.
-- Los [datos](https://github.com/jacarmona364/Riskalc/blob/Objetivo-0/Documentación%20Adicional/scoring.md) para resolver el problema han sido facilitados por la empresa en cuestión.
-
+- Los [datos](https://github.com/jacarmona364/Riskalc/blob/main/Documentación%20Adicional/scoring.md) para resolver el problema han sido facilitados por la empresa en cuestión.
 
+## Planificación del proyecto
+- [Documentación](https://github.com/jacarmona364/Riskalc/tree/Objetivo-1/docs)
```

Comment:
Estás enlazando un fichero en una rama. La rama deberías haberla fusionado y borrado. Debéis entender que las ramas se crean de forma transitoria para examinar los cambios por una tercera persona.